README.md

```diff
@@ -28,12 +28,15 @@ Add to your dependencies
 
     ALGOLIA_APPLICATION_ID=YOUR_APPLICATION_ID
     ALGOLIA_API_KEY=YOUR_API_KEY
+    ALGOLIA_HACKNEY_POOL_NAME=YOUR_HACKNEY_POOL_NAME
 
 #### Using config:
 
     config :algolia,
       application_id: YOUR_APPLICATION_ID,
-      api_key: YOUR_API_KEY
+      api_key: YOUR_API_KEY,
+      hackney_pool_name: ALGOLIA_HACKNEY_POOL_NAME
+
```

Comment:
Can we add a disclaimer regarding the requirement of starting the hackney pool in case the client wants to apply some custom configuration?